src/macis/gf/bandlan.cxx

```diff
@@ -293,57 +159,42 @@ bool GetEigsysBand(std::vector<std::vector<double> > &mat, int nSupDiag,
   eigvals.clear();
```

Comment:
This is a good piece of functionality. It might make sense to wrap it up into a nice LAPACK-style function for use elsewhere.